src/.clang-format

```diff
@@ -25,53 +25,74 @@ IndentWidth: 4
 UseTab: Never
 TabWidth: 8
 AccessModifierOffset: -4
-ConstructorInitializerIndentWidth: 4
-ContinuationIndentWidth: 4
+ConstructorInitializerIndentWidth: 0
```

Comment:
Actually in that PR was commit: https://github.com/Mudlet/Mudlet/commit/c7e21b6c9c375f7453fb589b38dd8fbf66ef8e23 where I had some revisions which got subsequently backed out again but one of them was a `ColumnLimit:120` - down from the prior 200... and a few settings that we had not set/(changed from defaults?)